### PR TITLE
Fix golint failures on cmd/kubeadm/app/util/config

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fix golint failures on cmd/kubeadm/app/util/config

**Which issue(s) this PR fixes**:
Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by yagonobre — COMMENTED** [review]
Thanks @SataQiu
/priority backlog
/lgtm

**Review by rosti — APPROVED** [review]
Thanks for your contribution @SataQiu !
/lgtm

File: cmd/kubeadm/app/util/config/initconfiguration_test.go [hunk 1]
```diff
@@ -34,16 +34,16 @@ import (
...
-	master_v1alpha3YAMLNonLinux  = "testdata/conversion/master/v1alpha3_non_linux.yaml"
-	master_v1beta1YAML           = "testdata/conversion/master/v1beta1.yaml"
-	master_v1beta1YAMLNonLinux   = "testdata/conversion/master/v1beta1_non_linux.yaml"
-	master_internalYAML          = "testdata/conversion/master/internal.yaml"
-	master_internalYAMLNonLinux  = "testdata/conversion/master/internal_non_linux.yaml"
-	master_incompleteYAML        = "testdata/defaulting/master/incomplete.yaml"
-	master_defaultedYAML         = "testdata/defaulting/master/defaulted.yaml"
-	master_defaultedYAMLNonLinux = "testdata/defaulting/master/defaulted_non_linux.yaml"
-	master_invalidYAML           = "testdata/validation/invalid_mastercfg.yaml"
+	masterV1alpha3YAML          = "testdata/conversion/master/v1alpha3.yaml"
```
Comment (neolit123) [inline cmd/kubeadm/app/util/config/initconfiguration_test.go]: in a follow up PR we should rename `master` and the folder names to to `controlplane` or something similar.

**Review by neolit123 — APPROVED** [review]
/approve

File: cmd/kubeadm/app/util/config/initconfiguration_test.go (hunk 1 above)
Comment (SataQiu) [reply cmd/kubeadm/app/util/config/initconfiguration_test.go]: @neolit123 Thanks! I will pay attention to the PRs in the future.

**Comment by neolit123** [comment]
@SataQiu 
if you have more similar changes planned that fix golint problems and touch the `hack` folder please send them in a single big PR. we don't want to ping the `hack` owners for multiple small changes.

thank you.

**Comment by SataQiu** [comment]
@neolit123 Thank you for your reminding! I'm sorry to bother the `hack` owners.

**Comment by fejta** [comment]
> we don't want to ping the `hack` owners for multiple small changes

Thanks Lubomir for the concern here :) And SataQui thanks for cleaning up these golint failures.

FWIW I'm happy to review any PR that removes lines from the `hack/.golint_failures` exception list!

/approve

**Pull request merged**